ui/src/common/ArtistShow.js

```diff
@@ -0,0 +1,350 @@
+          boxShadow: '-10px 0px 18px 5px black',
+        },
+        '& .MuiButton-label': {
+          color: `${theme.palette.primary.main}`,
+        },
+      },
+    },
+    album: {
+      marginBottom: '1em',
+    },
+  }),
+  { name: 'NDArtistPage' }
+)
+
+function ImgMediaCard({ artId, artist }) {
+  const [artisteInfo, setartisteInfo] = useState()
+  const [expanded, setExpanded] = useState(false)
+
+  let title = artist?.artist
+  let lastLink = ''
+  const link = artisteInfo?.biography?.match(
+    /<a\s+(?:[^>]*?\s+)?href=(["'])(.*?)\1/
+  )
+  const biography = artisteInfo?.biography?.replace(new RegExp('<.*>', 'g'), '')
+
+  const handleExpandClick = useCallback(() => {
+    setExpanded(!expanded)
+  }, [expanded, setExpanded])
+
+  useEffect(() => {
+    subsonic
+      .getArtistInfo(artId)
+      .then((resp) => resp.json['subsonic-response'])
+      .then((data) => {
+        if (data.status === 'ok') {
+          setartisteInfo(data.artistInfo)
+        }
+      })
+      .catch((e) => {
+        console.error('error on artist page', e)
+        return (
+          <Redirect
+            to={`/album?filter={"artist_id":"${artId}"}&order=ASC&sort=maxYear&
+              displayedFilters={"compilation":true}`}
+          />
+        )
+      })
+  }, [artId, artist])
+
+  if (link) {
+    lastLink = link[2]
+  }
+
+  const img = artisteInfo?.largeImageUrl
+  const classes = useStyles({ img, link, expanded })
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.bgContainer}>
+          <Card className={classes.martImage}>
+            <CardMedia
+              className={classes.cover}
+              image={`${artisteInfo?.mediumImageUrl}`}
+              title={title}
+            />
+          </Card>
+          <div className={classes.mdetails}>
+            <Typography component="h5" variant="h5">
+              {title}
+            </Typography>
+          </div>
+          <Card className={classes.artDetail}>
+            <Card className={classes.artImage}>
+              <CardMedia
+                className={classes.cover}
+                image={`${artisteInfo?.mediumImageUrl}`}
+                title={title}
+              />
+            </Card>
+            <div className={classes.details}>
+              <CardContent className={classes.content}>
+                <Typography component="h5" variant="h5">
+                  {title}
+                </Typography>
+                <Collapse
+                  collapsedHeight={'4.5em'}
+                  in={expanded}
+                  timeout={'auto'}
+                >
+                  <Typography variant={'body1'} onClick={handleExpandClick}>
+                    {biography}
+                    <Link
+                      href={lastLink}
+                      style={{ margin: '1px' }}
+                      target="_blank"
+                      rel="nofollow"
+                    >
+                      Read more...
```

Comment:
We discussed that this should follow the pattern in the `AlbumDetails` comment field. Also, if we were to go ahead with this approach, this would need to be in the translation file `en.json`

Reply:
Thank you for the review @deluan, actually, this `Read More...` is for the LastFM's link it's not for expanding bio. Should I add a `Read More...` key in `en.json`?

Reply:
Yes, I think we should. What about the "More"  bellow?